sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -1616,11 +1616,10 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
         Seq(tbl, ext_tbl).foreach { tblName =>
           sql(s"INSERT INTO $tblName VALUES (1, 'a', '2019-12-13')")
 
-          val expectedSize = 690
           // analyze table
           sql(s"ANALYZE TABLE $tblName COMPUTE STATISTICS NOSCAN")
           var tableStats = getTableStats(tblName)
-          assert(tableStats.sizeInBytes == expectedSize)
+          val expectedSize = tableStats.sizeInBytes
```

Comment:
Well, this is a removal of test coverage technically, @pan3793 .

This test case is a known issue which fails due to the Parquet metadata (mostly version string) change. However, I'd like not to remove this test coverage.

Reply:
I read the original PR, the intention of this test is to make sure partition stats get updated, even though it equals existing table stats. The number of table's sizeInBytes does not really matter here.

Generally, asserting the size of binary data files like Parquet/ORC does not make sense, it can vary due to metadata change, as you pointed out, this is likely caused by the version string change, and also might be affected by the compression codec, the compression data length might be different in different snappy version or platform.